webpack.config.production.js

```diff
@@ -0,0 +1,33 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+
+  output: {
+    filename: 'static/bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/'
+  },
+
+  devtool: 'source-map',
+
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        use: [
+          'babel-loader'
+        ],
+        exclude: /node_modules/
+      }
+    ]
+  },
+
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
```

Comment:
This is redundant since `webpack -p` will process output via uglify without the plugin configured.

Reply:
or we can use https://github.com/webpack-contrib/terser-webpack-plugin